docs/csharp/language-reference/keywords/using-statement.md

```diff
@@ -39,7 +39,9 @@ Provides a convenient syntax that ensures the correct use of <xref:System.IDispo
 ## Remarks  
  <xref:System.IO.File> and <xref:System.Drawing.Font> are examples of managed types that access unmanaged resources (in this case file handles and device contexts). There are many other kinds of unmanaged resources and class library types that encapsulate them. All such types must implement the <xref:System.IDisposable> interface.  
   
- As a rule, when you use an `IDisposable` object, you should declare and instantiate it in a `using` statement. The `using` statement calls the <xref:System.IDisposable.Dispose%2A> method on the object in the correct way, and (when you use it as shown earlier) it also causes the object itself to go out of scope as soon as <xref:System.IDisposable.Dispose%2A> is called. Within the `using` block, the object is read-only and cannot be modified or reassigned.  
+ Often, when you use an `IDisposable` object, you should declare and instantiate it in a `using` statement. The `using` statement calls the <xref:System.IDisposable.Dispose%2A> method on the object in the correct way, and (when you use it as shown earlier) it also causes the object itself to go out of scope as soon as <xref:System.IDisposable.Dispose%2A> is called. Within the `using` block, the object is read-only and cannot be modified or reassigned.
+ 
+ Note however some `IDisposable` objects are designed for re-use in which case it may be inefficent to create and dispose a new instance within a method call which `using` necessitates. For example an instance <xref:System.Net.Http.HttpClient> can be re-used multiple times without creating and disposing an instance every time. In such cases take care to correctly dispose the instance since `using` cannot be used.
```

Comment:
I agree with @markmnl concern that prescribing the `using` statement is a bit strong. They key thing with `IDisposable` is that the the party creating the instance should call call `Dispose` once it's done using the object.

There are really two scenarios:

1. The object is used only in the context of a method body. In that cause, the developer should be leveraging the `using` statement to ensure `Dispose` is properly called, including in error conditions.

2. The lifetime of the created object is longer exceeded the current method. In that case, the object should be a field or added to some collection and disposed later. In most cases, this means that the "owner" of the object needs to implement `IDisposable` too and dispose the objects that it holds.

The proposed wording from @markmnl doesn't express that point well; it almost sounds like that sharing implies not calling dispose.